src/blog/article/OtherArticlesHeader.js

```diff
@@ -9,7 +9,7 @@ import {
 const OtherArticlesHeader = () => (
   <Header>
     <Title>You may also like</Title>
-    <Link>
+    <Link href="/news-and-views">
```

Comment:
Awesome! Could we also update this component to be a Gatsby Link, as it's an internal link? We should only be using `a` tags for external links.

Reply:
@SofiaFredriksson This is now correct in that it's a Gatsby Link, however it is now no longer styled. The update to make it a Gatsby Link should not be here, but rather in the previously referenced component. 

In addition, the prop name isn't quite right for a relative link. `href` denotes an external link still. Take a look at the [docs](https://www.gatsbyjs.com/docs/gatsby-link/#replace-a-tags-with-the-link-tag-for-local-links).